src/css/leftpanel.scss

```diff
@@ -3,12 +3,12 @@
 .left-panel {
   z-index: 1;
   position: absolute;
-  top: 200px;
+  top: 140px;
```

Comment:
`Change Request` ALERT!! No more `bottom` and we want `height` back in there somehow